mbf_costmap_nav/src/mbf_costmap_nav/costmap_navigation_server.cpp

```diff
@@ -86,6 +87,12 @@ CostmapNavigationServer::CostmapNavigationServer(const TFPtr &tf_listener_ptr) :
   // start all action servers
   startActionServers();
 
+  // publish ready state
```

Comment:
this looks like it's being called from the constructor. should we publish this after the constructed has finished (maybe the last line?) or even in its own function entirely so this object has finished construction when it signals it's ready?

Reply:
done